apps/faqs/models.py

```diff
@@ -41,7 +41,7 @@ def __unicode__(self):
         return self.title
 
     class Meta:
-        verbose_name = 'catgeory'
+        verbose_name = 'categeory'
```

Comment:
You don't actually need this line at all, the model is named `Category` so this value will be correct. It's just the pluralised form which would be wrong (but that's being sorted on the line below).